packages/corvid-cli/test/setupJest.js

```diff
@@ -1 +1 @@
-jest.setTimeout(20000);
+jest.setTimeout(25000);
```

Comment:
Do we need it?

Reply:
Sure, it doesn't pass windows test with 20 secs